lib/vets-api/setups/hybrid.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require 'fileutils'
+
+module VetsApi
+  module Setups
```

Comment:
Yes, I was making a `bin/diagnosis` to test the setup. I abandoned it because I didn't think it was useful local development. However we could do something like `VetsApi::Setups::CI` with a "verification step" for the CI